cli/start.go

```diff
@@ -264,11 +265,18 @@ func runStart(_ *cobra.Command, _ []string) error {
 
 // rerunBackground restarts the current process in the background.
 func rerunBackground() error {
-	args := append([]string(nil), os.Args...)
-	// TODO(bdarnell): this looks silly in ps. Try to remove the
-	// `--background` flag instead of adding a second flag to reverse
-	// it.
-	args = append(args, "--background=false")
+	args := make([]string, 0, len(os.Args))
+	foundBackground := false
+	for _, arg := range os.Args {
+		if arg == "--background" || strings.HasPrefix(arg, "--background=") {
+			foundBackground = true
+			continue
+		}
+		args = append(args, arg)
+	}
+	if !foundBackground {
+		args = append(args, "--background=false")
```

Comment:
if `--background` wasn't specified on the commandline, it means the default is true somehow, so we need to override it with false.